tox.ini

```diff
@@ -17,17 +15,17 @@ test-executable =
 setenv =
     PYTHONPATH = {toxinidir}:{toxinidir}/tests
 commands =
-    dj{17,18,19,110,dev}: {[testenv]test-executable} test -p '*' core.tests --settings=settings_core
-    dj{17,18,19,110,dev}: {[testenv]test-executable} test basic.tests --settings=settings_basic
-    dj{17,18,19,110,dev}: {[testenv]test-executable} test related_resource.tests --settings=settings_related
-    dj{17,18,19,110,dev}: {[testenv]test-executable} test alphanumeric.tests --settings=settings_alphanumeric
-    dj{17,18,19,110,dev}: {[testenv]test-executable} test authorization.tests --settings=settings_authorization
-    dj{17,18,19,110,dev}: {[testenv]test-executable} test content_gfk.tests --settings=settings_content_gfk
-    dj{17,18,19,110,dev}: {[testenv]test-executable} test customuser.tests --settings=settings_customuser
-    dj{17,18,19,110,dev}: {[testenv]test-executable} test namespaced.tests --settings=settings_namespaced
-    dj{17,18,19,110,dev}: {[testenv]test-executable} test slashless.tests --settings=settings_slashless
-    dj{17,18,19,110,dev}: {[testenv]test-executable} test validation.tests --settings=settings_validation
-    dj{17,18,19,110,dev}: {[testenv]test-executable} test gis.tests --settings=settings_gis_spatialite
+    dj{18,111,dev}: {[testenv]test-executable} test -p '*' core.tests --settings=settings_core
```

Comment:
Inexplicably, the `-t core` option isn't needed for these tests, only those run locally.

We also can probably remove all of the manage_foo.py files and replace them with just manage.py and the `--settings=settings_foo` option, both here and in run_all_tests.sh